### DISCUSSION TITLE
Immowelt/Immonet not working at the moment?

### DISCUSSION THREAD

**Discussion by 0a1b** [orangecoding/fredy#115] [General]
Hi there,
Am I the only one who doesn't get listings from Immowelt and Immonet even with antscrape?
Greetz

**Comment by orangecoding** [comment]
Hey. No you are not the only one. It's a freakin cats and mice game. Im fixing this when I have some time

**Comment by 0a1b** [comment]
Thank you so much for your work! Best wishes!

**Comment by DerThermo** [comment]
Hi @orangecoding ,

I just wanted to share some additional information regarding this issue. Immowelt  was actually working (for me) as of December 6th, but since then, it has stopped functioning again. Thank you so much in advance for looking into this!

Greetz!
DerThermo

**Reply by orangecoding** [reply]
Immowelt works again. It was a problem due to a broken selector
